_includes/footer.html

```diff
@@ -0,0 +1,39 @@
+                        <a href="https://github.com/vacp2p" target="_blank"><svg width="60" height="59" viewBox="0 0 60 59" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M20.0685 47.1048C20.0685 47.3468 19.7903 47.5403 19.4395 47.5403C19.0403 47.5766 18.7621 47.3831 18.7621 47.1048C18.7621 46.8629 19.0403 46.6694 19.3911 46.6694C19.754 46.6331 20.0685 46.8266 20.0685 47.1048ZM16.3065 46.5605C16.2218 46.8024 16.4637 47.0807 16.8266 47.1532C17.1411 47.2742 17.504 47.1532 17.5766 46.9113C17.6492 46.6694 17.4194 46.3911 17.0565 46.2823C16.7419 46.1976 16.3911 46.3186 16.3065 46.5605ZM21.6532 46.3549C21.3024 46.4395 21.0605 46.6694 21.0968 46.9476C21.1331 47.1895 21.4476 47.3468 21.8105 47.2621C22.1613 47.1774 22.4032 46.9476 22.3669 46.7057C22.3306 46.4758 22.004 46.3186 21.6532 46.3549ZM29.6129 0C12.8347 0 0 12.7379 0 29.5161C0 42.9315 8.44355 54.4113 20.504 58.4516C22.0524 58.7299 22.5968 57.7742 22.5968 56.9879C22.5968 56.2379 22.5605 52.1008 22.5605 49.5605C22.5605 49.5605 14.0927 51.375 12.3145 45.9557C12.3145 45.9557 10.9355 42.4355 8.95161 41.5282C8.95161 41.5282 6.18145 39.629 9.14516 39.6653C9.14516 39.6653 12.1573 39.9073 13.8145 42.7863C16.4637 47.4557 20.9032 46.1129 22.6331 45.3145C22.9113 43.379 23.6976 42.0363 24.5685 41.2379C17.8065 40.4879 10.9839 39.5081 10.9839 27.871C10.9839 24.5444 11.9032 22.875 13.8387 20.746C13.5242 19.9597 12.496 16.7177 14.1532 12.5323C16.6815 11.746 22.5 15.7984 22.5 15.7984C24.9194 15.121 27.5202 14.7702 30.0968 14.7702C32.6734 14.7702 35.2742 15.121 37.6935 15.7984C37.6935 15.7984 43.5121 11.7339 46.0403 12.5323C47.6976 16.7298 46.6694 19.9597 46.3548 20.746C48.2903 22.8871 49.4758 24.5565 49.4758 27.871C49.4758 39.5444 42.3508 40.4758 35.5887 41.2379C36.7016 42.1936 37.6452 44.0081 37.6452 46.8508C37.6452 50.9274 37.6089 55.9718 37.6089 56.9637C37.6089 57.75 38.1653 58.7057 39.7016 58.4274C51.7984 54.4113 60 42.9315 60 29.5161C60 12.7379 46.3911 0 29.6129 0ZM11.7581 41.7218C11.6008 41.8428 11.6371 42.121 11.8427 42.3508C12.0363 42.5444 12.3145 42.629 12.4718 42.4718C12.629 42.3508 12.5927 42.0726 12.3871 41.8428C12.1935 41.6492 11.9153 41.5645 11.7581 41.7218ZM10.4516 40.7419C10.3669 40.8992 10.4879 41.0928 10.7298 41.2137C10.9234 41.3347 11.1653 41.2984 11.25 41.129C11.3347 40.9718 11.2137 40.7782 10.9718 40.6573C10.7298 40.5847 10.5363 40.621 10.4516 40.7419ZM14.371 45.0484C14.1774 45.2057 14.25 45.5686 14.5282 45.7984C14.8065 46.0766 15.1573 46.1129 15.3145 45.9194C15.4718 45.7621 15.3992 45.3992 15.1573 45.1694C14.8911 44.8911 14.5282 44.8548 14.371 45.0484ZM12.9919 43.2702C12.7984 43.3911 12.7984 43.7057 12.9919 43.9839C13.1855 44.2621 13.5121 44.3831 13.6694 44.2621C13.8629 44.1048 13.8629 43.7903 13.6694 43.5121C13.5 43.2339 13.1855 43.1129 12.9919 43.2702Z" fill="#121212"/></svg></a>
+                        <p><a href="https://github.com/vacp2p" target="_blank">Check out the code</a></p>
+                        <p>It all lives in Github. Here, test it out, XYZ</p>
```

Comment:
Replace with

Specs, research, proof of concepts. Check it out!

Reply:
Done